cloudigrade/api/models.py

```diff
@@ -875,8 +875,8 @@ class InstanceDefinition(BaseModel):
     instance_type = models.CharField(
         max_length=256, null=False, blank=False, db_index=True
     )
-    memory = models.IntegerField(default=0)
-    vcpu = models.IntegerField(default=0)
+    memory = models.DecimalField(max_digits=32, decimal_places=2, default=0.00)
+    vcpu = models.DecimalField(max_digits=32, decimal_places=2, default=0.00)
```

Comment:
Out of curiosity, have you seen actual fractional CPU (and memory, I suppose) values come out of the Azure API? Or is this driven because their API docs specify these as float/decimal precision values?

Reply:
Yea, Azure has 0.5 and such values. Worse yet, AWS reports memory in MB, while Azure in GB. We're storing them in the same field.

Reply:
We have an aws/azure flag so we can change how we interpret those values during calculation but that feels…. not great.

Reply:
Yikes! I thought I remembered AWS reporting in GB, but I just checked some real data and confirmed that, yes, the values we stored here are in MB.

This definitely sounds like an area we should refactor before too long. Maybe we could rename the vague `memory` to include units (`memory_mb`?) and multiply/divide as needed from the cloud-specific versions. Would you add a `# TODO` comment somewhere nearby to indicate that suggestion for future us? Or are you feeling ambitious enough to tackle that issue now?

Reply:
Honestly, this will bite us if we let it slip for too long. I'm going to go ahead and make everything one unit so we don't end up ruining any future reports.

Reply:
fyi re: vcpu decimals
<img width="761" alt="image" src="https://user-images.githubusercontent.com/34514071/124161351-281b8e00-da52-11eb-92cf-398bb175a050.png">